R/count_teleconnections.R

```diff
@@ -133,6 +137,21 @@ count_watershed_teleconnections <- function(data_dir,
 
         distance_values %>% mean()/m_to_km -> avg_withdr_dis_km
 
+        # Get number of cities
+        watershed_usage_table %>%
+          subset(Watershed_DVSN_ID %in% city_intake_ids) -> city_usage
+        if(nrow(city_usage) == 0){
+          dependent_population <- 0
+          n_other_cities <- 0
+        }else{
+        city_usage[,4] %>% unique() %>%
+          left_join(cities_population, by = "city_state") %>%
+          .[["Population"]] %>%
+          sum() -> dependent_population
```

Comment:
Just to confirm... this means that dependent population will include the main city, right? It's fine either way, just want to make sure what figure is returned.

Reply:
@swd-turner When the `get_watershed_usage()` function is used it filters out the main city before it loads in the table. That table is used to look at the rest of the cities. So we have a column that has the main city's population, and then we have a column for the dependent population of other cities. So it does not include the main city. I can easily change that if you want it that way though!